### DISCUSSION TITLE
请问在使用语音合成，动态图推理，怎样去调整合成语音的语速和音量呢？

### DISCUSSION THREAD

**Discussion by thehzzz** [PaddlePaddle/PaddleSpeech#2556] [Q&A] [answered]
https://github.com/PaddlePaddle/PaddleSpeech/blob/develop/demos/text_to_speech/README_cn.md
请问在使用语音合成，动态图推理，怎样去调整合成语音的语速和音量呢？

**Comment by yt605155624** [comment, marked as answer]
可以用后处理 https://github.com/PaddlePaddle/PaddleSpeech/issues/2383
